src/docs/self-hosted/releases.mdx

```diff
@@ -20,7 +20,11 @@ Sentry cuts regular releases for self-hosting to keep it as close to [sentry.io]
 
 We encourage everyone to regularly update their Sentry installations to get the best and the most recent Sentry experience.
 
-To upgrade, all you need to do is download or check out the version of onpremise repository you want, replace your existing folder's contents with that, and then run `./install.sh`. **We may have some updated configuration, especially for new features, so always check the example config files under the [sentry directory](https://github.com/getsentry/onpremise/blob/master/sentry/) and see if you need to update your existing configuration**. We do our best to automate critical configuration updates, but you should always check your configs during upgrades.
+To upgrade, all you need to do is download or check out the version of onpremise repository you want, replace your existing folder's contents with that, and then run `./install.sh`.
+
+<Alert title="Configuration Updates" level="info">
+We may have some updated configuration, especially for new features, so always check the example config files under the [sentry directory](https://github.com/getsentry/onpremise/blob/master/sentry/) and see if you need to update your existing configuration. We do our best to automate critical configuration updates, but you should always check your configs during upgrades.
```

Comment:
Need to use HTML for link instead of Markdown here.

<img width="723" alt="Screen Shot 2021-08-17 at 11 24 03 AM" src="https://user-images.githubusercontent.com/134455/129754543-de2f8b70-b5a1-4681-a841-176041e88727.png">